internal/executor/daemon/executor_test.go

```diff
@@ -0,0 +1,278 @@
+package daemon
+
+import (
+	"errors"
+	"io/ioutil"
+	"octavius/internal/executor/client"
+	"octavius/internal/executor/config"
+	"octavius/internal/pkg/constant"
+	"octavius/internal/pkg/kubernetes"
+	"octavius/internal/pkg/log"
+	executorCPproto "octavius/internal/pkg/protofiles/executor_cp"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	v1 "k8s.io/api/core/v1"
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func init() {
+	log.Init("info", "", false,1)
```

Comment:
add space after comma, `log.Init("info", "", false, 1)` do you run go fmt or no?